code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -560,6 +560,9 @@
 
 /obj/item/weapon/rig/equipped(mob/living/carbon/human/M)
 	..()
+	if (is_held())
```

Comment:
This fixes a case where you could remove a rig, and the pieces (helmet, boots, etc) would remain on your body while you hold the control module in your hand.

Same bug existed for voidsuits and is fixed below too